starter-project-backend/src/app.ts

```diff
@@ -11,5 +11,6 @@ app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 
 app.use("/", json({}))
+app.use("/api/articles/:articleId/ratings", routes.ratingsRouter)
 
```

Comment:
I thought '/api/articles/' was for articlesRouting. And using '/api/ratings/' also didn't make sense since ratings are dependent on articles. I think it is better if we address this issue in Phase-2 - Interlink between models.

Reply:
it can be /api/rate then inside rating.ts use /:articleId/ratings to get article id